src/disc/disc.py

```diff
@@ -11,6 +11,44 @@
 ADMINISTRATOR = 1 << 3
 VIEW_CHANNEL = 1 << 10
 
+class User:
+    def __init__(self, id: str, username: str, discriminator: str, global_name: str=None, avatar: str=None) -> None:
+        self.id = id
+        self.username = username
+        self.discriminator = discriminator
```

Comment:
what's a discriminator? I wasn't able to figure it out by looking at where `discriminator` is used, so let's add a short comment.

Reply:
Discriminators were deprecated, they used to be the #xxxx after a username. Accounts which haven't migrated over yet have them still.

Reply:
ahhh, I didn't know those were called discriminators, cool! Def add a lil comment. Glad we're handling them!